example/src/VideoroomContext.tsx

```diff
@@ -48,6 +52,8 @@ const VideoroomContextProvider = (props) => {
   const [currentCamera, setCurrentCamera] = useState<CaptureDevice | null>(
     null
   );
+  const [isDevMode, setIsDevMode] = useState(false);
```

Comment:
can we save it, so that we don't need to turn it on again after refresh?

Reply:
I can use `AsyncStorage` lib for that, I wasn't sure whether it's worth the extra effort. If you prefer I can add it

Reply:
I think adding async storage lib not a big deal, we're probably going to need it in the future anyway. And it will be useful, I forgot to turn on the dev mode a couple of times already.